### DISCUSSION TITLE
Configuration problem  - data directory is required\

### DISCUSSION THREAD

**Discussion by dtwarogpl** [mxpv/podsync#466] [General]
### Discussed in https://github.com/mxpv/podsync/discussions/364

<div type='discussions-op-text'>

<sup>Originally posted by **dtwarogpl** July 25, 2022</sup>
Hello, Im trying to run podsync from docker container. Have tried almost everything but Im, not able to. 

I get same error every time:
`time="2022-07-25T12:45:54Z" level=fatal msg="failed to load configuration file" error="1 error occurred:\n\t* data directory is required\n\n"`

My docker-compose file:
`version: '2.2'
services:
  podsync:
    container_name: podsync
    image: mxpv/podsync:latest
    restart: always
    ports:
      - 8081:80
    volumes:
      - ./data:/app/data/
      - ./config.toml:/app/config.toml
`

my configuration:
`# Configure where to store the episode data
[storage]
  type = "local"
  [storage.local]
  data_dir = "/app/data"
[tokens]
youtube = "AIzaSyARXXXXXXXXXXX"

[feeds]
    [feeds.ID1]
    url = "https://www.youtube.com/channel/UCIXXXXX"
    page_size = 5
    format = "audio"
    clean = { keep_last = 10 }`

Appreciate any help </div>

**Comment by mickremy** [comment]
I had the same issue and solved it by changing the config.toml 

Remove this :
```
[storage]
type = "local"
[storage.local]
data_dir = "/app/data"
```

And replace it by just this :
```
[server]
port = 8080
hostname = "http://10.8.0.1:8080"
data_dir = "/app/data" 
```
